### PR TITLE
Update Terraform and plugins

### PR DESCRIPTION
**Pull request by amarthadan**
8ac64293116e3d46a54f17462ddfe066eb307158 taken from https://github.com/api3dao/airnode/pull/1659

### CODE REVIEW

File: packages/airnode-deployer/terraform/gcp/main.tf [hunk 1]
```diff
@@ -56,18 +56,20 @@ module "run" {
...
+  source_dir         = var.handler_dir
+  memory_size        = 512
+  timeout            = 65
+  configuration_file = var.configuration_file
+  secrets_file       = var.secrets_file
+  region             = var.gcp_region
+  project            = var.gcp_project
+  schedule_interval  = 1
+  max_instances      = var.disable_concurrency_reservation ? null : 1
+  invoke_targets = {
```
Comment (Siegrift) [inline packages/airnode-deployer/terraform/gcp/main.tf:70]: TBH: This formatting looks broken

File: packages/airnode-deployer/terraform/gcp/providers.tf [hunk 2]
```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 4.7"
+      version = "~> 4.54"
```
Comment (Siegrift) [inline packages/airnode-deployer/terraform/gcp/providers.tf:5]: Downgrade?

Comment (amarthadan) [reply packages/airnode-deployer/terraform/gcp/providers.tf:5]: I thought so as well at the first glance but it's actually from `7` to `54`. It's not `5.4`.

File: packages/airnode-deployer/terraform/gcp/main.tf (hunk 1 above)
Comment (amarthadan) [reply packages/airnode-deployer/terraform/gcp/main.tf:70]: I agree. But it's how the `terraform fmt` thinks it's best :man_shrugging: I can add an empty line above `invoke_targets` if that helps.

Comment (Siegrift) [reply packages/airnode-deployer/terraform/gcp/main.tf:70]: I don't care that much, it's just that it formatted the whole section for nothing (nothign changed there)

**Review by Siegrift — APPROVED** [review]
👍 LGTM

**Pull request merged**